.circleci/config.yml

```diff
@@ -132,68 +126,6 @@ jobs:
             - oppia_tools/
       - upload_screenshots
```

Comment:
Why uplaod screenshots for running typescript checks?

Reply:
Removed it, as well as the `persist_to_workspace` step.